utils/scripts/ocm/ocm.py

```diff
@@ -844,6 +848,71 @@ def update_notification_email_address(self, addon_name, email_address, exit_on_f
             else:
                 return False
 
+    def get_latest_ods_candidate_version(self, x_stream, y_stream):
+        """
+        get the latest  candidate version
+        Args:
+            x_stream:
+            y_stream:
+
+        example 4.8 = x_stream.y_stream
+        """
+        cmd = ("ocm list versions --channel-group  candidate |"
```

Comment:
make "candidate" as a default parameter, so that this function will be generic for all channel groups.

Reply:
okay sure, you can take this in next PR, please try using regex for matching different string formats